make-based/build-all.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# https://stackoverflow.com/a/246128/4804196
+SOURCE=${BASH_SOURCE[0]}
+while [ -L "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+    DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+    SOURCE=$(readlink "$SOURCE")
+    [[ $SOURCE != /* ]] && SOURCE=$DIR/$SOURCE # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+SCRIPT_DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+
+top=$(pwd)/../workdir
+libs=$(pwd)/../workdir/libs
+uk=$(pwd)/../workdir/unikraft
+
+source "$SCRIPT_DIR"/../include/common_functions
+
+if test $# -lt 1; then
+    echo -e "Usage: $0 <PR-number | branch> [deps]\n" 1>&2
+    echo "The dependencies are PRs from the unikraft core or external libraries repositories." 1>&2
+    echo -e "\ne.g. $0 123 musl/10 unikraft/20 lwip/15" 1>&2
+    echo "will pull PRs 123 and 20 from the unikraft core, PR 10 from lib-musl and PR 20 form lwip."
+    echo -e "\n$0 staging musl/10 unikraft/20 lwip/15" 1>&2
+    echo "will pull PR 20 from the unikraft core, PR 10 from lib-musl and PR 20 form lwip."
+    exit 1
+fi
+
+log_file=$(pwd)/logs/err.log
+PR_number="$1"
+
+unset UK_ROOT UK_WORKDIR UK_LIBS
+mkdir "$(pwd)/logs" 2> /dev/null
+
+rm -rf "$top"
```

Comment:
```suggestion
```

You don't want to remove items that you've set up in the past (this just happened to me).